src/ExceptionTransformer.ts

```diff
@@ -46,9 +48,80 @@ class ExceptionTransformer {
         exception.fallback_message
       );
     }
-
     return exceptionMap;
   }
+
+  public generateSpesificFieldError(errorInfo: Exception | null | undefined) {
+    const detail =
+      errorInfo && typeof errorInfo.detail === "object" && errorInfo.detail;
```

Comment:
This expression is being used multiple times. Maybe you can create an util for this.